src/frontend/src/modules/cart/CartAdditionalItem.vue

```diff
@@ -40,18 +40,12 @@ export default {
   data() {
     return {
       counter: 0,
-      additionalSum: 0
     }
   },
   methods: {
     sendCount(counter) {
       this.counter = counter;
```

Comment:
внутреннее состояние не нужно. принимай counter в качестве пропса

Reply:
я так и делал поначалу, но там ошибка выходит: Avoid mutating a prop directly since the value will be overwritten whenever the parent component re-renders. Instead, use a data or computed property based on the prop's value. Prop being mutated: "counter"
поэтому я объявлял внутреннее состояние

Reply:
это было из-за того что ты пытался менять значение самого пропса. Через v-model или напрямую в методах. А так делать нельзя, потому что это значение "не принадлежит" этому компоненту. его хозяин где-то наверху и только он его может менять. поэтому мы просто кидаем наверх событие чтобы там его поймали и обновили значение

Reply:
если отдавать counter из корня страницы  в качестве пропса на все компоненты с счетчиком, то и меняться он будет у всех компонентов при нажатии на кнопку +, поэтому сделал локальное состояние

Reply:
Ну это естественно должно быть не одно состояние на все ингредиенты. Ты можешь хранить, например, в виде объекта вида `{ananas: 2, pepperoni: 0, ...}`

Reply:
все, разобрался, спасибо